src/components/landing-page/use-fetch-production.ts

```diff
@@ -2,20 +2,24 @@ import { useEffect, useState } from "react";
 import { API } from "../../api/api";
 import { TProduction } from "../production-line/types";
 
-type TUseFetchProduction = (id: number | null) => {
+type TUseFetchProduction = (
+  id: number | null,
+  refresh?: number
```

Comment:
I think handling refreshes is probably better done by having the caller set id to null and then the number to refresh.

E.g. once a production has successfully been fetched, the component using this hook should set the production id to null. When the production needs to be refreshed, set the production id to a number again.

In the ManageProductions component, this could be handled by introducing a separation between the from value for `productionId` that we want to remain, and a `productionIdToFetch` that can be freely toggled between a number and null to trigger re-fetching.